Add view/edit/save support for 1.10 conky confs also.

zcot opened the pull request:
These tweaks and additions affect 2 files: Main and EditWidget.

They provide:

1. recognition and loading of 1.10 confs also into the main widget list.
2. load/edit/save ability in the Edit widget panel on 3 tabs: Location, Size, and Transparency.

I've broken the commits into small sections so it should be easy to see each affect easily.

Vistaus committed:
Added Dutch translation

zcot committed:
Add recognition to load conky 1.10 conf files

zcot committed:
support load/edit/save conky of 1.10 conf

zcot committed:
change get_value funct to also read conky 1.10 conf

zcot committed:
get_value catches lua quoted vars. version switches for set_value

zcot committed:
fixup load widget for 1.10 confs

zcot committed:
option for miminum_height + minimum_width for 1.10 confs

zcot committed:
fixup apply-save widget for 1.10 confs

zcot committed:
version switch for window size for apply-save widget for 1.10 confs

zcot committed:
allow height padding READ to account for 1.10 conf lua text block

zcot committed:
add height-padding WRITE, for 1.10 conf lua text block

zcot committed:
adjust 'transparency' scheme for 1.10 lua true/false vars

zcot committed:
fix apply-save bug: write window_argb_value after 'transparency', so it is not overwritten.

zcot commented:
@teejee2008 I didn't mess with versioning or documentation not knowing how you might approach that, but perhaps an easier way would be for me to just manage a forked branch separately, bump the version or even rename, and then users could choose which version they would like. I'm open for any suggestion on that.

teejee2008 commented:
Hi @zcot, you can go ahead and fork this project if you wish to. I haven't worked on this application in a long time and I'm not getting time to support it. Please change the name in case you release a forked version. Send me a link when you release it so that I can use it too :)

zcot committed:
Add simple build instructions.

zcot committed:
Merge branch 'master' of https://github.com/Vistaus/conky-manager into Vistaus-master

Adding Dutch translation by Vistaus.

zcot committed:
Merge branch 'Vistaus-master' into add-1.10-support

Bring Vistaus translation.

zcot committed:
Transition to conky-manager2

zcot committed:
Continuing conky-manager2 transition

zcot committed:
fix bug in Croatian language header

zcot committed:
Continuing conky-manager2 transition

zcot committed:
Integration of Dutch translation by Vistaus

zcot committed:
Add vanity label :p

zcot committed:
Add vanity label correction :D

zcot committed:
Icons get transitioned to conky-manager2

zcot committed:
Updated info

zcot committed:
Use .md format now

zcot committed:
Update build info

zcot committed:
Update build info for macOS

zcot commented:
@teejee2008,

Here is fork conky-manager2: https://github.com/zcot/conky-manager2

I just picked up with continuation. The version is 2.5(from previous 2.4).

zcot

tioguda committed:
Add Portuguese translation from Brazil

Commit:
update-default-themes-NVIDIA-for-BOM

zcot committed:
Merge pull request #5 from tioguda/add-1.10-support

Add Portuguese translation from Brazil

zcot committed:
Merge pull request #6 from jamartin9/remove-nvidia-bom

update-default-themes-NVIDIA-for-BOM

zcot committed:
Add ability to import various non ConkyManagerTheme(cmtp.7z) designed archives using existing Import function

zcot committed:
add tioguda credit for Portuguese Brazil translation

zcot committed:
credit THANKS for jamartin9 remove-nvidia-bom marker from nvidia theme

zcot committed:
bump version to 2.6

tioguda committed:
Integrate Portuguese translation from Brazil

zcot committed:
Merge pull request #7 from tioguda/add-1.10-support

Integrate Portuguese translation from Brazil

tioguda committed:
Update Brazilian Portuguese translation

zcot committed:
Merge pull request #8 from tioguda/add-1.10-support

Update Brazilian Portuguese translation

horgan committed:
Removed negative time value

zcot committed:
add 'if_up' and 'addr' to network edit widget

zcot committed:
Merge branch 'add-1.10-support' of https://github.com/zcot/conky-manager2 into add-1.10-support

zcot committed:
Merge pull request #13 from horgan/patch-2

Removed negative time value

fehlix committed:
added French translations

fehlix committed:
merged French translations from default branch

fehlix committed:
adjust lua format detection and desktop-session in startup

renyhp committed:
update requirements

zcot committed:
Merge pull request #17 from renyhp/patch-1

update requirements, add gettext system for Ubuntu variation.

afelinczak committed:
Support meson (#19)

* configuring vscode

* init meson support works with vs code plugins and builds using ninja - to be tested on ppa

* remove po files with non standard names

* debian build tested

freddii committed:
added german translation (#22)

* added german translation

* some better german translations

* some more better german translations

Co-authored-by: freddii <https://freddii@github.com>

zcot committed:
clean old po design, add credits

zcot committed:
credit

zcot committed:
Merge pull request #16 from fehlix/add-1.10-support

added French translations, adjusted lua-format detection, lua_load timeout changes and desktop-session into startup.sh

zcot committed:
clean up translation, credit

zcot committed:
remove old format naming translation files

zcot committed:
french

zcot committed:
to version 2.72

zcot committed:
allow conky or conky-all dependency

zcot committed:
allow other conky package dependencies

yeji0407 committed:
added korean translation

mok0 committed:
Fixed debian control files for lintian errors

- Fix various errors in debian/control
- Add man page
- Install appstream metadata in correct directory
- Fix errors in desktop entry
- donate.svg file is not an executable

mok0 committed:
Clean up ConkyManager2 icon

- Remove spurious white/grey pixels inside CM lettering
- Lower "2" slightly and darken hue from pure yellow for better harmony with
  tomato color of logo.

zcot committed:
Merge pull request #25 from yeji0407/add-1.10-support

added korean translation

zcot committed:
Merge pull request #26 from mok0/fix-debian-control

Fix debian control files to satisfy lintian errors.

zcot committed:
Small updates, Korean translation, debian packaging changes, icon artwork. version 2.73

zcot committed:
implement man

zcot committed:
For debugging configs, implement Open In Terminal

zcot committed:
multiple adjustments

zcot committed:
use xdg feature instead of exo for editor

zcot committed:
convert original themepack to 1.10 lua configs

zcot committed:
get the lunch money

zcot committed:
allow null DESKTOP_SESSION

zcot committed:
add teejee2008 autoupdate-ppa information

zcot committed:
add .deb file instruction

zcot committed:
fix default themepack autoinstall

chmodmasx committed:
spanish lang added (#49)

* spanish lang added

* added spanish lang to makefile
